code/modules/jobs/jobs.dm

```diff
@@ -228,7 +228,18 @@ GLOBAL_LIST_INIT(security_positions_hud, list(
 	JOB_HUD_DEPUTY,
 	JOB_HUD_RAWSECURITY))
 
+/// Put any removed jobs here so they can still show in playtime listings.
+GLOBAL_LIST_INIT(exp_removed_jobs, list(
+//	"Virologist",
+))
+GLOBAL_PROTECT(exp_removed_jobs)
 
+/// Put any removed jobs here so they can still show in playtime listings.
+GLOBAL_LIST_INIT(exp_removed_jobsmap, list(
+//	EXP_TYPE_CREW = list("titles" = list("Virologist")),
+//	EXP_TYPE_MEDICAL = list("titles" = list("Virologist")),
```

Comment:
Debug code? (Viro is going to get removed anyways so it's fine)

Reply:
It's commented out to show the correct usage of the array. Otherwise it would not be clear.